mainlyetcetera/rotten_tomatillos
# Feature/user error messages

## Is this a fix or a feature?
- feature
## What is the change/fix?
- add error messages for both the whole movie collection and individual movie details
  - these appear if the api calls fail
- refactor to use an `ErrorMsg` component
- start adding styling to the `ErrorMsg` component 
  - note: this still needs work!
- add link to the error messages allowing the user to return to the main movies view
## Where should the reviewer start?
- `src/components/ErrorMsg.js`
## How should this be tested?
- pull down branch (not to main)
- `git checkout` to test branch
- run `npm start`
- go into code and modify url's so that they will fail
- observe rudimentary error messages
## This PR is related to which issues:
closes #30 #28
